jabgui/src/main/java/module-info.java

```diff
@@ -190,5 +190,6 @@
     // requires mslinks;
     requires org.antlr.antlr4.runtime;
     requires org.libreoffice.uno;
+    requires java.scripting;
```

Comment:
You do know what "alphabetically" means?

<img width="635" height="352" alt="image" src="https://github.com/user-attachments/assets/c94e5646-63e5-459b-8d54-5c621b3860af" />

Reply:
https://github.com/JabRef/jabref/pull/14478/files shows it.

I assume IntelliJ might have added it - maybe, you can just remove it?